packages/fuselage/src/components/Tabs/Tabs.tsx

```diff
@@ -15,5 +14,3 @@ export function Tabs({ children, divider = true, ...props }: TabsProps) {
     </Box>
   );
 }
-
-Tabs.Item = TabsItem;
```

Comment:
right now this will break all its usage. 
i suggest keeping it there and adding the deprecated sign